File: Unlucky Days.py
import datetime as dt
import calendar
def unlucky_days(year):
    count = 0
    for month in range(1, 13):
        date = dt.date(year, month, 13)
        youbi = calendar.day_abbr[date.weekday()]
        if youbi=="Fri":
            count+=1
    return count

File: Unlucky Days.ts
export function unluckyDays(year: number): number {
  let counter = 0;
  for (let i = 1; i <= 12; i++) {
      let d = new Date(i + "/13/" + year);                          
      if (d.getDay() == 5) {
          counter++;
      }
  }
  return counter;
}
